test/data-tests.jsx

```diff
@@ -108,6 +112,36 @@ b.test("DataTests", {
     // TODO: initial function removal in items?
 
 
+  },
+
+  testIsText: function(b) {
+    var doc = b.doc();
+
+    var tf = b.text(b.LOREM, 0, 0, 200, 200);
+
+    assert(b.isText(b.characters(tf)));
+    assert(b.isText(b.characters(tf)[0]));
+
+    assert(b.isText(tf.insertionPoints));
+    assert(b.isText(tf.insertionPoints.firstItem()));
+
+    assert(b.isText(b.words(tf)));
+    assert(b.isText(b.words(tf).firstItem()));
+
+    assert(b.isText(b.lines(tf)));
+    assert(b.isText(b.lines(tf).firstItem()));
+
+    assert(b.isText(tf.textStyleRanges));
+    assert(b.isText(tf.textStyleRanges.firstItem()));
+
+    assert(b.isText(b.paragraphs(tf)));
+    assert(b.isText(b.paragraphs(tf).firstItem()));
+
+    assert(b.isText(tf.textColumns));
+    assert(b.isText(tf.textColumns.firstItem()));
+
+    // anyone know if and how an [object Texts] can be created?
```

Comment:
Hm, you can create a `Text` object by using `itemByRange()` (as I did in line 144). These text objects all don't have add methods, yet you can create (or rather retrieve) most of them. However, I still see no way to create a `Texts` object. If you don't know of a way either, I think I should take it out from the checks again. Otherwise it would mean an unnecessary check for every checked object.
So, please don't merge yet.